web/modules/custom/dpl_recommender/dpl_recommender.info.yml

```diff
@@ -0,0 +1,10 @@
+name: DPL recommender
+description: recommender component
+package: DPL
+dependencies:
+  - drupal:block
+  - dpl_fbs:dpl_fbs
+  - dpl_publizon:dpl_publizon
```

Comment:
Do this module depend on FBS/Publizon modules? I think is would be dependent on some search stuff in dpl_react_apps?

Reply:
I dont think so...